.github/workflows/kind-e2e.yaml

```diff
@@ -63,6 +64,12 @@ jobs:
             channel-auth-scenario: "SASL_SSL"
           - test-suite: ./test/e2e_new
             channel-auth-scenario: "SASL_PLAIN"
+          - test-suite: ./test/experimental
+            channel-auth-scenario: "SSL"
+          - test-suite: ./test/experimental
+            channel-auth-scenario: "SASL_SSL"
+          - test-suite: ./test/experimental
+            channel-auth-scenario: "SASL_PLAIN"
```

Comment:
These are only for testing KafkaChannel with auth. The reasoning is:
- We cannot define individual KafkaChannels to use auth as it is a global thing
- Thus, we re-run the entire KafkaChannel reconciler test suite in a separate job with auth enabled.

You can remove these lines here.

Reply:
agreed, and yes - we have upstream... also several jobs

Reply:
@aliok @matzew I'm not sure I follow. What I'm doing here is excluding running the experimental features suite as part of the KafkaChannel e2e jobs. This insures that for the SSL, SASL_SSL & SASL_PLAIN scenarios (i.e. the 3 KafkaChannel scenarios) only the test-suite `./test/e2e_channel` will be picked up, and `./test/experimental` will be excluded.

Reply:
Context FTW :smile: 

looking _at_ the file directly, I do see `exclude` a couple of lines above.

Reply:
> looking at the file directly, I do see exclude a couple of lines above.

My bad. I am sorry 🤦